ext/standard/tests/general_functions/phpinfo.phpt

```diff
@@ -29,6 +29,7 @@ PHP Extension => %d
 Zend Extension => %d
 Zend Extension Build => API%s
 PHP Extension Build => API%s
+Integer Size => %d-bit
```

Comment:
Would it make sense to maybe call it `PHP Integer Size`. It could be a bit confusing for people familiar with C.

Reply:
And also should the result be more `32 bits` or `64 bits` instead. The 64-bit is more for architucture identification.